bin/configs/java-wiremock.yaml

```diff
@@ -0,0 +1,7 @@
+generatorName: java-wiremock
+outputDir: samples/server/petstore/java-wiremock
```

Comment:
can you please add this new folder to the .github/workflows/samples-java* files so that CI will test it moving forward?